### PR TITLE
#102 Projects section on the site

### PR DESCRIPTION
**Pull request by articobandurini**
resolves https://github.com/leapdao/leapdao-website/issues/102

### CODE REVIEW

**Comment by troggy** [comment]
peep.cash is missing

**Comment by troggy** [comment]
Logo for Planet A doesn't look good on a white background. And it isn't a logo really — too noisy, too little contrast. AFAIR there was no "official" logo for the project, so I suggest to remove the "logo" for it altogether — use a transparent placeholder — until it has one.

**Comment by troggy** [comment]
a bit inconsistent
image

File: index.html [hunk 1]
```diff
@@ -170,6 +170,60 @@ <h4>Security in mind</h4>
...
+              <p>
+                <img src="/img/logo-deora.png" alt="" /><br />
+                <b>Deora</b> – decentralized quadratic voting solution for improved democracy.
+                <br /><a href="https://www.deora.earth/">Deora website</a>
+              </p>
+            </div>
+            <div class="col">
+              <p>
+                <img src="/img/logo-motion.png" alt="" /><br />
+                <b>Motion Protocol</b> – blockchain ecosystem for all transactions arising in film.
```
Comment (troggy) [inline index.html]: > blockchain ecosystem for all transactions arising in film.

this doesn't sound correct. Could you please double check all the projects description with peeps on Slack ?

Comment (articobandurini) [reply index.html]: FYI this description is taken directly from the https://www.motion.ooo/ website, see section "WHAT IS MOTION PROTOCOL?"

**Comment by articobandurini** [comment]
> a bit inconsistent
> image

do you think that transparent placeholder will look better?
any useful suggestions?

**Comment by troggy** [comment]
summoning Webmaster 😄

**Comment by articobandurini** [comment]
@sunify ?

**Comment by MaxStalker** [comment]
@articobandurini I think what @troggy meant is that all sections, apart from `Planet A` have the a project name as paragraph opener. So making it this way could make it a bit more consistent :)
### PlanetA
**Planet A** - social game that simulates Tragedy of the...

**Comment by MaxStalker** [comment]
I would also recommend to replace Peep.cash logo. That shadow doesn't work on white :)
This one should work better: 
peep cash logo

**Comment by troggy** [comment]
half-assed a "logo" for planet A. Feel free to discard my proposal commit

image

**Comment by articobandurini** [comment]
@troggy what about now?

**Comment by johannbarbie** [comment]
we should remove motion from partners, if they are a Dapp

File: index.html [hunk 2]
```diff
@@ -170,6 +170,60 @@ <h4>Security in mind</h4>
...
+              <p>
+                <img src="/img/logo-deora.png" alt="" /><br />
+                <b>Deora</b> – decentralized quadratic voting solution for improved democracy.
+                <br /><a href="https://www.deora.earth/">Deora website</a>
+              </p>
+            </div>
+            <div class="col">
+              <p>
+                <img src="/img/logo-planeta.png" alt="" /><br />
+                <b>Planet A</b> – social game that simulates Tragedy of the CO₂mmons from eco perspective.
```
[the remark this answers is not in the document]
Comment (articobandurini) [reply index.html]: Too long, also not sure why we have to advertise SDP right here - there should be a blog post for that. Word "serious" is added though.

**Comment by articobandurini** [comment]
OK peeps, are we good to go with this PR now? :)

**Pull request merged**